.github/workflows/ci.yaml

```
@@ -104,6 +104,8 @@ env:
     test_unpack
     test_weakref
     test_yield_from
+  # Python version targeted by the CI.
+  PYTHON_VERSION: 3.11.4
```

When we revert this to major.minor, this should most likely be surrounded by quotation marks. Otherwise, we'd be eventually actually targeting Python 3.2 when we meant to target Python 3.20.

```suggestion
  PYTHON_VERSION: "3.11.4"
```

---

good point and I was recently reading about this too.